sootup.core/src/main/java/sootup/core/graph/BackwardsStmtGraph.java

```diff
@@ -27,15 +27,19 @@
 import sootup.core.jimple.common.stmt.Stmt;
 
 /** @author Zun Wang */
-public class BackwardsStmtGraph extends ForwardingStmtGraph {
+public class BackwardsStmtGraph<V extends BasicBlock<V>> extends ForwardingStmtGraph<V> {
 
-  public BackwardsStmtGraph(@Nonnull StmtGraph<?> stmtGraph) {
+  public BackwardsStmtGraph(@Nonnull StmtGraph<V> stmtGraph) {
     super(stmtGraph);
   }
 
   @Override
   public Stmt getStartingStmt() {
-    throw new UnsupportedOperationException();
+    List<Stmt> startStmts = getStartingStmts();
+    if (startStmts.isEmpty()) {
+      return null;
+    }
+    return startStmts.get(0);
```

Comment:
these should return the tails of the control flow graph - which could be multiple elements

Reply:
Got it